docs/openDAQ-streaming.md

```diff
@@ -158,8 +158,11 @@ This allows the client to know when all signals are described and data is to be
 ### Value Index
 An uint64 sent with each value of an implicit signal. It is used to align the value of the implicit signal with the table steps.
 
+### Related Signals
+A signal may have related signals that carry additional information. Examples are domain signal and status signal.
+Each relation is described by the id of the related signal and the type of the related signal (i.e. "domain", "status").
 
-
+Only signals that are already subscribed may be used as related signals!

```

Comment:
Just a quick question—what happens if the ID of a signal that hasn't been subscribed to yet is listed as related? Is there any validation in place that triggers an error if the check fails?

Reply:
Since the consumer code does only collect the information and does not do anything with it, there is no validation yet.